### PR TITLE
Handlebars for templates and custom email template

### PR DESCRIPTION
**Pull request by korycins**
- use handlebars instead of django template
- convert templates to use handlebars
- extend admin email plugin to provide input for custom subjects and templates 🍾 🍾 🍾 
- tests

### CODE REVIEW

File: saleor/plugins/email_common.py [hunk 1]
```diff
@@ -105,12 +115,54 @@ class EmailConfig:
 }
 
 
-def send_email(config: EmailConfig, recipient_list, template_name, context):
+def block_trans(this, options, *args, **kwargs):
+    # TODO add translation here
```
Comment (fowczarek) [inline saleor/plugins/email_common.py:119]: Should we fix it before merge to the feature branch or it should be separate PR?

**Review by fowczarek — APPROVED** [review]

File: saleor/plugins/email_common.py (hunk 1 above)
Comment (korycins) [reply saleor/plugins/email_common.py:119]: Not in this PR. I need to figure out the best way for this.

**Review by IKarbowiak — APPROVED** [review]

**Review by patrys — CHANGES_REQUESTED** [review]
Please make LGTM happy. It shows 13 function calls using an unsupported argument.

**Comment by korycins** [comment]
@patrys  Yes, I am aware, it is related with the changes for user_email_plugin. I was plingin to fix it in second PR where I will use the correct function calls.
